src/components/Discussion/index.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import {
+  DiscussionBackground,
+  DiscussionContainer,
+  DiscussionTextContainer
+} from "../../StyledComponents";
+
+const Discussion = () => (
+  <DiscussionBackground>
+    <DiscussionContainer>
+      <h3>THERE IS ALWAYS ROOM FOR DISCUSSION</h3>
+      <DiscussionTextContainer>
+        <p style={{ marginTop: "0" }}>
```

Comment:
did you mean to have this inline style? we have removed it almost everywhere else

Reply:
replaced with a className.